src/Magnum/LibOVRIntegration/Test/IntegrationTest.cpp

```diff
@@ -0,0 +1,209 @@
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2015
+              Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+#include <Corrade/TestSuite/Tester.h>
+
+#include "Magnum/Magnum.h"
+#include "Magnum/LibOvrIntegration/Conversion.h"
+#include "Magnum/Math/Matrix3.h"
+#include "Magnum/Math/DualQuaternion.h"
+
+namespace Magnum { namespace LibOvrIntegration { namespace Test {
+
+// define missing == operators for LibOVR types
+bool operator== (const ovrSizei& a, const ovrSizei& b);
+bool operator== (const ovrVector2i& a, const ovrVector2i& b);
+bool operator== (const ovrVector2f& a, const ovrVector2f& b);
+bool operator== (const ovrVector3f& a, const ovrVector3f& b);
+bool operator== (const ovrRecti& a, const ovrRecti& b);
+bool operator== (const ovrMatrix4f& a, const ovrMatrix4f& b);
+bool operator== (const ovrQuatf& a, const ovrQuatf& b);
+bool operator== (const ovrPosef& a, const ovrPosef& b);
```

Comment:
If you wrap the operators in an anonymous namespace, these forward declarations won't be needed (If I'm guessing correctly, you added them to avoid compiler warning, right?).